### ISSUE TITLE
Rule Indent fail with jsdoc @param property

### ISSUE THREAD

**Issue by sarvaje** [eslint/eslint#9150]
**Tell us about your environment**

* **ESLint Version:**
4.5.0
* **Node Version:**
8.3.0
* **npm Version:**
5.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
    "rules": {
    "parser": "typescript-eslint-parser",
    "indent": ["error", 4, {
            "SwitchCase": 1,
            "VariableDeclarator": 1
        }]
}
```
</details>

**What did you do? Please include the actual source code causing the issue.**

```js
/**
 * test
 * @param {string} param - description
 */
const newFunc = (param) => {
    return param;
};
```

**What did you expect to happen?**
Nothing (this is what I receive if I use `indent-legacy`)
**What actually happened? Please include the actual, raw output from ESLint.**
  3:1  error  Expected indentation of 0 spaces but found 3  indent

**Comment by not-an-aardvark** [comment]
Thanks for reporting. However, I can't seem to reproduce this issue -- no error is reported in the [demo](https://eslint.org/demo):

![](https://i.gyazo.com/e4f99309d5c9a6e565271f56a30db9d3.png)

Are you sure you pasted your code and configuration correctly?

**Comment by sarvaje** [comment]
Sorry @not-an-aardvark. I take a deeper look, and the problem is when I have configured the parser `typescript-eslint-parser`. I've updated the configuration.

Should I open the issue in the `typescript-eslint-parser` repo?

**Comment by not-an-aardvark** [comment]
Yes, it's possible that this is a bug in `typescript-eslint-parser`. Thanks!

**Comment by sarvaje** [comment]
ok. Thanks!

**Comment by not-an-aardvark** [comment]
Sounds good. I'm going to close this because I don't think it's an issue with ESLint itself.
